addons/html_editor/static/src/core/history_plugin.js

```diff
@@ -367,6 +367,9 @@ export class HistoryPlugin extends Plugin {
     }
     stageSelection() {
         const selection = this.shared.getEditableSelection();
+        if (this.currentStep.mutations.length >= 1) {
```

Comment:
There are now many tests where we have mutations when we call stageSelection.
There is a least one case I have in the top of my head where this will cause a problem:
- set the selection in an empty p
- move the selection somewhere
=> now there is the hint plugin that add a attribute mutation, so not staging of selection will work afterwards
- type a letter
- undo
=> the selection is in the empty P rather than the place before undo.